tools/add_outline.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+"""
+Add black outline to all sprites in a folder
+"""
+import os
+import glob
+import argparse
+import numpy as np
+from PIL import Image
+
+BLANK = np.array([0, 0, 0, 0])
+BLACK = np.array([0, 0, 0, 255])
+
+
+def is_outline(table: np.array, col: int, row: int) -> bool:
+    """
+    Determine if the point belongs to the outline
+    """
+    up = table[row - 1, col, :]
+    down = table[row + 1, col, :]
+    left = table[row, col - 1, :]
+    right = table[row, col + 1, :]
+
+    for side in [left, right, up, down]:
+        if (side != BLACK).any() & (side != BLANK).any():
+            return True
+
+    return False
+
+
+def draw_outline(data: np.array) -> None:
+    """
+    Add outline to the pixels array in-place
+    """
+    for i in range(1, 30):
+        for j in range(1, 30):
+            if (data[j, i, :] == BLANK).any():
+                if is_outline(data, i, j):
+                    data[j, i, :] = BLACK
+
+
+def main(args):
```

Comment:
```suggestion
def main(args: argparse.Namespace) -> None:
```
type hinting the main